x/slashing/keeper/infractions.go

```diff
@@ -18,9 +18,6 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr cryptotypes.Addre
 
 	// fetch the validator public key
 	consAddr := sdk.ConsAddress(addr)
-	if _, err := k.GetPubkey(ctx, addr); err != nil {
-		panic(fmt.Sprintf("Validator consensus-address %s not found", consAddr))
-	}
```

Comment:
can you touch on why this is being removed

Reply:
@vuong177 can.  We are getting on a flight right now, so probably tomorrow